src/mcts/search.cc

```diff
@@ -118,18 +119,21 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
   common_info.nps =
       common_info.time ? (total_playouts_ * 1000 / common_info.time) : 0;
   common_info.tb_hits = tb_hits_.load(std::memory_order_acquire);
-
   int multipv = 0;
   for (const auto& edge : edges) {
+    float score = (edge.HasNode() && edge.node()->GetParent())
```

Comment:
It's guaranteed that edge->node will have parent, and more of that, it's guaranteed that it's `root_node_` (as we just called GetBestChildrenNoTemperature for it)